content/200-concepts/100-components/03-prisma-migrate/050-prisma-migrate-flows.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: 'Migration flows'
+metaTitle: 'Migration flows (Concepts)'
+metaDescription: 'Prisma Migrate is a declarative data modeling and schema migration tool that is available via the Prisma CLI.'
+---
+
+<TopBlock>
+
+</TopBlock>
+
+## Modifying migration behavior
+
+Prisma Migrate cannot automatically detect that you want to _rename_ a field - instead, it will add a new field and remove the old one. To rename a field (for example, `name` &rarr; `fullname`) and maintain data, you must create and modify a **draft migration**:
+
+1. Generate draft
+2. Edit draft
+3. Run draft
+
+## Collaborating with a team
+
+To bring in changes to the database schema and migrations from collaborators, pull the changed  Prisma Schema and the new migration files and use `prisma migrate dev` to apply the changes.
+
+![https://s3-us-west-2.amazonaws.com/secure.notion-static.com/8bdb1aad-8801-418a-8723-b41e315706dd/New_Migrate_flows_-_Development_workflow_with_collaborators.jpg](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/8bdb1aad-8801-418a-8723-b41e315706dd/New_Migrate_flows_-_Development_workflow_with_collaborators.jpg)
+
+When calling `prisma migrate dev`, Prisma Migrate will check first if there are any migrations that are yet to be applied and applies them. After applying these pending migrations, it will look for changes in the Prisma schema, and if there are any, create new migrations and apply them.
```

Comment:
```suggestion
When calling `prisma migrate dev --preview-feature`, Prisma Migrate will check first if there are any migrations that are yet to be applied and applies them. After applying these pending migrations, it will look for changes in the Prisma schema, and if there are any, create new migrations and apply them.
```
Should we add the --preview-feature flag here?